### DISCUSSION TITLE
Cloud Gaming's future

### DISCUSSION THREAD

**Discussion by boson13579** [1101-GS3073-C/Python-Community-GS3073-C#466] [General]
[Google is shutting down Stadia](https://www.theverge.com/2022/9/29/23378713/google-stadia-shutting-down-game-streaming-january-2023)

google stadia is a cloud gaming service started from 2019, and it will be stoped in 2023. What does this mean to cloud gaming. How will Geforce now(another cloud gaming service powered by Nvdia) will be. Let's us wait and see.

**Comment by brian431** [comment]
> [Google is shutting down Stadia](https://www.theverge.com/2022/9/29/23378713/google-stadia-shutting-down-game-streaming-january-2023)
> 
> 
> 
> google stadia is a cloud gaming service start from 2019, and it will stop in 2023. What does this mean to cloud gaming. How will Geforce now(another cloud gaming service powered by Nvdia) will be. Let's us wait and see.

Google stadia 的失敗我認為不能代表整個cloud gaming的失敗。Microsoft 及Nvidia在這部分都做的不錯，且隨著網路傳輸速度越來越快，應該還是有不錯的前景。

**Comment by RogelioKG** [comment]
可惜，我認為是他們預設的目標客群失準了，
平台本身沒有足夠的遊戲，
但卻還要付費+月費才能遊玩，
這讓許多輕度玩家望而卻步，
而核心玩家也早有設備可以遊玩了。
目前雲端遊戲最大痛點還是延遲，
不過我也同樣相信未來能逐步克服這些困難。

**Comment by awkward-willy** [comment]
許多遊戲玩家可能因為設備等因素導致沒有辦法好好體驗完整遊戲內容，因此出現了雲端遊戲的服務。雖受限於網速，但概念上是相當不錯的。Geforce now 也有看到有人評價還不錯，應該是完整的商業模式以及顧客的觀念等部分還有待開發。

**Comment by Yilin20021016** [comment]
雖然說是很好用 但是格價對於我來說不太友好
